example/src/examples/Animations/AnimatedLineOffsets.tsx

```diff
@@ -0,0 +1,256 @@
+                });
+              }}
+            />
+            <Button
+              title={'Zero offsets'}
+              onPress={() => {
+                startOffset.current = 0;
+                endOffset.current = 0;
+                animator.setLineString({
+                  coordinates: buildRandomizedLine(),
+                  startOffset: startOffset.current,
+                  endOffset: endOffset.current,
+                });
+                setStartOffsetState(startOffset.current);
+                setEndOffsetState(endOffset.current);
+              }}
+            />
+          </View>
+        </View>
+
+        <Divider style={{ marginVertical: 5 }} />
+
+        <View>
+          <View style={rowStyle}>
+            <Text>{'Start Offset'}</Text>
+            <Text>{startOffsetState.toFixed(2)} m</Text>
+          </View>
+          <Slider
+            {...sliderProps}
+            value={startOffsetState / lineLength}
+            onSlidingComplete={(v) => {
+              startOffset.current = v * lineLength;
+              animator.setStartOffset({
+                offset: startOffset.current,
+                durationMs: duration.current,
+              });
+              setStartOffsetState(startOffset.current);
+            }}
+          />
+        </View>
+
+        <Divider style={{ marginVertical: 5 }} />
+
+        <View>
+          <View style={rowStyle}>
+            <Text>{'End Offset'}</Text>
+            <Text>{endOffsetState.toFixed(2)} m</Text>
+          </View>
+          <Slider
+            {...sliderProps}
+            value={endOffsetState / lineLength}
+            onSlidingComplete={(v) => {
+              endOffset.current = v * lineLength;
+              animator.setEndOffset({
+                offset: endOffset.current,
+                durationMs: duration.current,
+              });
+              setEndOffsetState(endOffset.current);
+            }}
+          />
+        </View>
+
+        <Divider style={{ marginVertical: 5 }} />
+
+        <View>
+          <View style={rowStyle}>
+            <Text>{'Duration'}</Text>
+            <Text>{(durationState / 1000).toFixed(2)} s</Text>
+          </View>
+          <Slider
+            {...sliderProps}
+            value={durationState / maxDuration}
+            onSlidingComplete={(v) => {
+              duration.current = v * maxDuration;
+              animator.setStartOffset({
+                offset: startOffset.current,
+                durationMs: duration.current,
+              });
+              animator.setEndOffset({
+                offset: endOffset.current,
+                durationMs: duration.current,
+              });
+              setDurationState(duration.current);
+            }}
+          />
+        </View>
+      </View>
+    );
+  }, [
+    startOffsetState,
+    endOffsetState,
+    durationState,
+    lineLength,
+    animator,
+    buildRandomizedLine,
+  ]);
+
+  return (
+    <Page {...props}>
```

Comment:
```suggestion
    <>
```

I'd like to remove  `Page` and `BaseExampleProps`.

Reply:
Oh - how do you want to go back from an example without `Page`?